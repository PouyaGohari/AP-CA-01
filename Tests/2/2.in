2
participant1 participant2
participant1:0 YES|1 NO
participant2:0 YES|1 NO