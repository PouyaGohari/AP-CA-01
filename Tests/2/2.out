Time slot 0:
YES: participant1 participant2 
NO: 
IF_NECESSARY: 
###
Time slot 1:
YES: 
NO: participant1 participant2 
IF_NECESSARY: 
###
